src/main/java/budgetbuddy/logic/parser/commandparsers/transactioncommandparsers/TransactionAddCommandParser.java

```diff
@@ -58,14 +58,11 @@ public TransactionAddCommand parse(String args) throws ParseException {
 
         Amount amount = CommandParserUtil.parseAmount(argMultiMap.getValue(PREFIX_AMOUNT).get());
 
-        Optional<String> optionalDescription = argMultiMap.getValue(PREFIX_DESCRIPTION);
-        Description description = optionalDescription.isPresent()
-                ? CommandParserUtil.parseDescription(optionalDescription.get())
-                : null;
+        Description description = CommandParserUtil.parseDescription(argMultiMap.getValue(PREFIX_DESCRIPTION).get());
 
-        Optional<String> optionalAccount = argMultiMap.getValue(PREFIX_ACCOUNT);
-        Account account = optionalAccount.isPresent()
-                ? CommandParserUtil.parseAccount(optionalAccount.get())
+        Optional<String> optionalAccountName = argMultiMap.getValue(PREFIX_ACCOUNT);
+        Name accountName = optionalAccountName.isPresent()
+                ? CommandParserUtil.parseAccountName(optionalAccountName.get())
                 : null;
```

Comment:
Could just do `parseName(optionalAccountName.get())`, but since the new `CommandParserUtil#parseAccountName` looks functionally identical to `CommandParserUtil#parseName` I guess it doesn't matter too much.